go/epochtime/tests/mock_tester.go

```diff
@@ -57,11 +57,14 @@ func EpochtimeSetableImplementationTest(t *testing.T, backend api.Backend) {
 func MustAdvanceEpoch(t *testing.T, backend api.SetableBackend, increment uint64) api.EpochTime {
 	require := require.New(t)
 
-	epoch, err := backend.GetEpoch(context.Background(), 0)
+	ctx, cancel := context.WithTimeout(context.Background(), recvTimeout)
```

Comment:
In case of bugs there can be timeouts here, yes. So I added this to fail early instead of blocking for 10m. Of course the timeouts should not happen now, only if something is messed up as it was during development.